src/legacy-styles/components/_editor-override.scss

```
@@ -29,11 +29,10 @@
 	.editor,
 	.journal-entry-content {
 		font-family: var(--font-editor);
-		max-width: max(90%, 60ch);
-		margin-inline: auto;
+		margin: 1rem;
```

This here will create double scroll bars in Journal Entries. I suggest you remove it or change it to:

```suggestion
	article.journal-entry-page.text {
		margin-inline: 1rem;
	}
```